test/e2e/servicebindingrequest_test.go

```diff
@@ -152,9 +169,9 @@ func assertAppDeployed(
 	if len(containers) != 1 {
 		return nil, fmt.Errorf("can't find a container in deployment-spec")
 	}
-	//if len(containers[0].EnvFrom) != 1 {
-	//	return nil, fmt.Errorf("can't find envFrom in first container")
-	//}
+	if len(containers[0].EnvFrom) > 0 {
+		return nil, fmt.Errorf("Containers  are less than 0")
```

Comment:
Is this message correct?  I believe it doesn't match with the conditional.

Reply:
Outdated